Framework/PythonInterface/mantid/plots/datafunctions.py

```diff
@@ -1041,4 +1042,8 @@ def update_colorbar_scale(figure, image, scale, vmin, vmax):
 
 
 def figure_contains_only_3d_plots(fig) -> bool:
```

Comment:
As we saw with determining whether figure was "empty" I'm not sure this is a check that we can confidently put in a general low-level place like this and have it always be correct. As we only use it once in the workbench can we keep it local. Should we also use the similar check that is used in the delete handle of the figuremanager with the Axes rather than specifically checking for a `QuadMesh`?